packages/webpack-config/package.json

```diff
@@ -26,18 +26,22 @@
     "prepack": "node ../../tools/clean-package-json.mjs"
   },
   "dependencies": {
-    "@pandell/postcss-config": "workspace:packages/postcss-config",
+    "@babel/core": "^7.27.1",
```

Comment:
❓`@babel/core` / `babel-loader` / `ts-loader` add a lot of noise to `nuget_modules` (and they are not needed to build `webpackConfig.ts`). What do you think about making them peer dependencies (just like `webpack`)?

Reply:
Moved to peer dependencies. I don't need `@babel/core` then because we aren't directly using `babel-loader` anymore.